x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/components/fleet_server_on_prem_instructions.tsx

```diff
@@ -288,8 +289,8 @@ export const useFleetServerInstructions = (policyId?: string) => {
   }, [notifications.toasts]);
 
   const refresh = useCallback(() => {
-    return Promise.all([outputsRequest.resendRequest(), refreshSettings()]);
-  }, [outputsRequest, refreshSettings]);
+    return Promise.all([refreshOutputs, refreshSettings()]);
```

Comment:
@afgomez It seems that `refreshOutputs` is no longer called? Is that correct/intentional? `Promise.all` expects an array of promises, so a function, iiuc, seems incorrect

Reply:
Oh, yes! Good catch. Since this was merged I'll do a quick fixup